### PR TITLE
Update release notes and CHANGELOG based on 18.9 changes 

### PR DESCRIPTION
**Pull request by hafizrahman**
Update marketing release notes for 18.9 based on listed changes from devs.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This PR has the <code>Releases</code> label: some checks will be skipped.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <!--
 Contains editorialized release notes. Raw release notes should go into `RELEASE-NOTES.txt`.
 -->
+## 18.9
+Stay ahead with our latest WooCommerce update! Explore our new Watch App for instant store data and order updates at a glance. We've fixed the issue with shipping methods, so you can now create and update orders seamlessly. Plus, you can now view, add, edit, or remove multiple shipping lines on orders, enhancing your order management.
```
Comment (joshheald) [inline CHANGELOG.md]:
```suggestion
Stay ahead with our latest WooCommerce update! Explore our new Watch App for instant store data and order updates at a glance. We've fixed the issue with shipping methods, plus you can now have multiple shipping lines on orders, so you can handle complex shipping needs for your orders seamlessly.
```

File: WooCommerce/Resources/release_notes.txt [hunk 2]
```diff
@@ -1,5 +1 @@
-- [***] Explore our new Watch App to get glance of you store data and latest orders.
-- [*] Orders: Fixes an issue where adding shipping to an order without selecting a shipping method prevented the order from being created/updated. [https://github.com/woocommerce/woocommerce-ios/pull/12870]
-- [**] Orders: Multiple shipping lines can now be viewed, added, edited, or removed on orders; previously only the first shipping line on an order was supported. [https://github.com/woocommerce/woocommerce-ios/pull/12888]
-- [internal] Orders: An empty order list screen after applying filters no longer affects Dashboard cards eligibility. [https://github.com/woocommerce/woocommerce-ios/pull/12873]
-
+Stay ahead with our latest WooCommerce update! Explore our new Watch App for instant store data and order updates at a glance. We've fixed the issue with shipping methods, so you can now create and update orders seamlessly. Plus, you can now view, add, edit, or remove multiple shipping lines on orders, enhancing your order management.
```
Comment (joshheald) [inline WooCommerce/Resources/release_notes.txt]:
```suggestion
Stay ahead with our latest WooCommerce update! Explore our new Watch App for instant store data and order updates at a glance. We've fixed the issue with shipping methods, plus you can now have multiple shipping lines on orders, so you can handle complex shipping needs for your orders seamlessly.
```

**Review by joshheald — APPROVED** [review]
Fine as is, but I've tweaked the wording a bit. Feel free to ignore if you prefer the original, but it was a little wordy and repetitive for my liking.

**Comment by hafizrahman** [comment]
Thanks @joshheald, the sentence flows so much nicer now. 👍🏼

**Pull request merged**